nuxeo-services/login/nuxeo-platform-login-token-rest-api/pom.xml

```diff
@@ -0,0 +1,83 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.nuxeo.ecm.login</groupId>
+    <artifactId>parent</artifactId>
+    <version>8.3-SNAPSHOT</version>
+  </parent>
+  <artifactId>nuxeo-platform-login-token-rest-api</artifactId>
+  <name>Nuxeo Token Authentication Rest API</name>
+  <dependencies>
+    <dependency>
+      <groupId>org.nuxeo.ecm.platform</groupId>
+      <artifactId>nuxeo-platform-login-token</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>javax.servlet-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.nuxeo.ecm.automation</groupId>
+      <artifactId>nuxeo-automation-server</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.nuxeo.ecm.platform</groupId>
+      <artifactId>nuxeo-rest-api-server</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.nuxeo.ecm.platform</groupId>
+      <artifactId>nuxeo-rest-api-io</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+    </dependency>
+
+    <!-- Test -->
+    <dependency>
+      <groupId>org.nuxeo.ecm.core</groupId>
+      <artifactId>nuxeo-core-test</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.nuxeo.ecm.platform</groupId>
+      <artifactId>nuxeo-platform-test</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.nuxeo.runtime</groupId>
+      <artifactId>nuxeo-runtime-datasource</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.nuxeo.ecm.platform</groupId>
+      <artifactId>nuxeo-platform-types-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.nuxeo.ecm.platform</groupId>
+      <artifactId>nuxeo-platform-url-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.nuxeo.ecm.platform</groupId>
+      <artifactId>nuxeo-platform-tag-api</artifactId>
```

Comment:
👍

On 27 June 2016 at 11:23, Thomas Roger <notifications@github.com> wrote:

> In nuxeo-services/login/nuxeo-platform-login-token-rest-api/pom.xml
> <https://github.com/nuxeo/nuxeo/pull/135#discussion_r68546132>:
>
> > +      <artifactId>nuxeo-runtime-datasource</artifactId>
> > +      <scope>test</scope>
> > +    </dependency>
> > +    <dependency>
> > +      <groupId>org.nuxeo.ecm.platform</groupId>
> > +      <artifactId>nuxeo-platform-types-core</artifactId>
> > +      <scope>test</scope>
> > +    </dependency>
> > +    <dependency>
> > +      <groupId>org.nuxeo.ecm.platform</groupId>
> > +      <artifactId>nuxeo-platform-url-core</artifactId>
> > +      <scope>test</scope>
> > +    </dependency>
> > +    <dependency>
> > +      <groupId>org.nuxeo.ecm.platform</groupId>
> > +      <artifactId>nuxeo-platform-tag-api</artifactId>
>
> Do we really need test dependencies on tag?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/nuxeo/nuxeo/pull/135/files/12ccdaae8e71aa21090cbc66213b6367148d665f#r68546132>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AEYyC7vgHjrMmTMSGng2o4I7ZPcyKQbXks5qP5angaJpZM4I-5hP>
> .
>



-- 
Guillaume Renard, Dev Team, Nuxeo SA
Open Source ECM
Mail : grenard@nuxeo.com
Web : http:/www.nuxeo.com
Tél : +33 1 40 33 79 87